### DISCUSSION TITLE
Can I disable the completion of ":!" in cmdline?

### DISCUSSION THREAD

**Discussion by Dice21** [hrsh7th/nvim-cmp#1040] [Q&A] [answered]
It takes too much time to complete the **"!"** command on my device. 
So is there any way to disable it?

**Comment by Shougo** [comment, marked as answer]
Please see this.

https://github.com/hrsh7th/nvim-cmp/issues/821

**Reply by Dice21** [reply]
Thank you too much. Before I searched for "!" in Issues and got nothing.
Now i set `completion.keyword_length` to `3` and it works for me.
